### PR TITLE
Modifying docs according to changes in data-bootstrap endpoint. (AB-7…

### PR DESCRIPTION
**Pull request by yuriykurylenko**
…752, AB-7691)

https://autobutler.atlassian.net/browse/AB-7691
https://autobutler.atlassian.net/browse/AB-7752.

### CODE REVIEW

File: source/includes/_data_bootstrap.md [hunk 1]
```diff
@@ -78,10 +80,16 @@ makes.id                         | integer       | no            | The id of the
 makes.name                       | string        | no            | The name of the make (human readable)
 user                             | object        | no            | An object representing the details of the current workshop
 user.countryId                   | integer       | no            | The id of the country to which the workshop belongs
+user.countryCod                  | integer       | no            | The code of the country to which the workshop belongs
```
Comment (larssg) [inline source/includes/_data_bootstrap.md]: Typo

File: source/includes/_data_bootstrap.md [hunk 2]
```diff
@@ -41,10 +41,12 @@ jQuery.ajax({
   "makes": [],
   "user": {
     "countryId": 3,
+    "countryCode": 3,
```
Comment (dipth) [inline source/includes/_data_bootstrap.md]: This doesn't look like a real example of a country code...

File: source/includes/_data_bootstrap.md [hunk 3]
```diff
@@ -41,10 +41,12 @@ jQuery.ajax({
...
     "locale": "uk",
     "name": "Pauls Workshop",
     "requireAutomaticPayment": false,
-    "vatFactor": 0.2
+    "vatFactor": 0.2,
+    "subMechanics": []
```
Comment (dipth) [inline source/includes/_data_bootstrap.md]: Please add some examples of what some actual submechanics might look like in the response

File: source/includes/_data_bootstrap.md (hunk 1 above)
Comment (dipth) [inline source/includes/_data_bootstrap.md]: - You're missing an `e` at the end of `user.countryCod`
- Is it really an `integer` or is it in fact a string?

File: source/includes/_data_bootstrap.md [hunk 4]
```diff
@@ -78,10 +80,16 @@ makes.id                         | integer       | no            | The id of the
 makes.name                       | string        | no            | The name of the make (human readable)
 user                             | object        | no            | An object representing the details of the current workshop
 user.countryId                   | integer       | no            | The id of the country to which the workshop belongs
+user.countryCode                 | string        | no            | The code of the country to which the workshop belongs
 user.locale                      | string        | no            | The locale of the country to which the workshop belongs
 user.name                        | string        | no            | The name of the workshop
 user.requireAutomaticPayment     | boolean       | no            | Weather or not the workshop is required to have a payment method set up
 user.vatFactor                   | float         | no            | The default VAT factor of the workshop
+user.subMechanics                | array(object) | yes           | A list of sumbmechanics in case if current user is a supebidder
```
Comment (larssg) [inline source/includes/_data_bootstrap.md]: Couple of typos

File: source/includes/_data_bootstrap.md [hunk 5]
```diff
@@ -78,10 +80,16 @@ makes.id                         | integer       | no            | The id of the
...
+user.subMechanics.id             | integer       | no            | The id of a specific submechanic
+user.subMechanics.name           | string        | no            | The name of a specific submechanic
+user.subMechanics.zip            | string        | no            | ZIP code of the area where specific submechanic is located
```
Comment (larssg) [inline source/includes/_data_bootstrap.md]: I really hate that we call the zip codes :-(

**Pull request merged**